frontend/css/free_home_view.css

```diff
@@ -0,0 +1,73 @@
+.color-div {
+    background-color: white;
+}
+
+.color-center {
+    background-color: blue;
+}
+
+/* Extra small devices (portrait phones, less than 576px)*/
+@media (max-width: 575.98px) {
+    .test-div {
+        flex-basis: 0;
+    }
+
+    .test-center {
+        flex-basis: 100%;
+    }
+}
+
+/*Small devices (landscape phones, 576px and up)*/
+@media (min-width: 576px) and (max-width: 767.98px) {
+    .test-div {
+        flex-basis: 5%;
+    }
+
+    .test-center {
+        flex-basis: 90%;
+    }
+}
+
+/*Medium devices (tablets, 768px and up)*/
+@media (min-width: 768px) and (max-width: 991.98px) {
+    .test-div {
+        flex-basis: 10%;
+    }
+
+    .test-center {
+        flex-basis: 80%;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=6343113)